_documentation/general/beta.md

```diff
@@ -17,7 +17,9 @@ Every new version of AntennaPod is sent to our Beta Testers via the Google Play
 
 So you want to have the latest and greatest? That's certainly possible! Simply visit the AntennaPod listing in the Play store and go to "Become a beta tester" at the bottom of the page. Or, if you're at your computer, you can go to [this url](https://play.google.com/apps/testing/de.danoeh.antennapod).
 
-![Become a beta tester!](/assets/images/documentation/project-beta.png)
+<img src="/assets/images/documentation/project-beta.png" alt="Become a beta tester!" style="max-width:400px;">
```

Comment:
The website theme already sets the `max-width` to 100% of the page width. If you now set it to 400px, that is overwritten and can lead to an overflow on phones (see screenshot below when I just changed the CSS on the "Inspect element" page of the browser). Assigning the `width` CSS property directly would be more reliable. Then it is 400px by default but might also be smaller when the page is smaller.

![image](https://user-images.githubusercontent.com/5811634/115107479-54536280-9f6b-11eb-8e2e-53a68091ea5e.png)

Reply:
Whoops. Yes, thank you. Will update it.